### ISSUE TITLE
Problem with module version

### ISSUE THREAD

**Issue by jackaponte** [backdrop-contrib/coffee#1]
First--thanks for the port, Andy!

I recently noticed that the download link on the [Backdrop project page for Coffee](https://backdropcms.org/project/coffee) is broken. The release version listed there is 1.x-2.2, which matches the [tag in GitHub](https://github.com/backdrop-contrib/coffee/releases/tag/1.x-2.2). However, when you actually download the module, `coffee.info` says that it's version 1.x-1.1, which I'm guessing has something to do with the project page download link being broken!

**Comment by thekenshow** [comment]
The link on backdropcms.org is still broken for this.

**Comment by klonos** [comment]
Yep, https://github.com/backdrop-contrib/coffee/releases/download/1.x-2.2/coffee.zip gives a 404 on GitHub and if you look at the [2.2 release page](https://github.com/backdrop-contrib/coffee/releases/tag/1.x-2.2), you'll see that there is no coffee.zip file (only the source .zip and .tar.gz files).

So, it's not the project page link that's broken per se. It's https://github.com/backdrop-ops/backdropcms.org/issues/264

**Comment by klonos** [comment]
@biolithic, can you try to remove the release and tag another in its place? Perhaps this will be enough to trigger the creation of the coffee.zip file.

**Comment by simone960** [comment]
Any update on this ? I still can't install this module from the backdrop's backend Install modules page.

**Comment by izmeez** [comment]
The module does install from the `/admin/modules/install` page and the installed modules shows on the `admin/modules` list as `1.x-2.2.2`. The .info file shows `version = 1.x-1.1`

Meanwhile the release page is all mixed up, https://github.com/backdrop-contrib/coffee/releases/tag/1.x-2.2.2 with the title 1.x-1.2.2 but the asset as 2.2.2 (zip).

Maybe it has something to do with how the release was tagged?

**Comment by Wylbur** [comment]
Is this still an issue?  Reopen if needed!
